NuGet.Docs/Create/Hosting-Your-Own-NuGet-Feeds.md

```diff
@@ -65,31 +65,32 @@ You can also host a remote (or internal) feed on a server that runs IIS. There a
 1. NuGet.Server
 2. NuGet Gallery
 
-For relatively small projects with a small set of packages go with NuGet.Server, it is basically a view of a network share or local folder through http, and as such is easy to setup and works quite well when the number of packages is small. _However_ if the package count is high, say in the thousands, the server will take a very long time to boot. The alternative is to use the [NuGet Gallery Project](https://github.com/NuGet/NuGetGallery/wiki/Hosting-the-NuGet-Gallery-Locally-in-IIS) it is more complex to set up and host, but offers a lot more nuget.org like features.
+For relatively small projects with a small set of packages go with NuGet.Server, it is basically a view of a network share or local folder through http, and as such is easy to setup and works quite well when the number of packages is relatively small.
```

Comment:
with the new nuget.server this is not relevant. Can we add a blurb describing it?